<launch>

 <!-- forest joy Node-->
  <!--node pkg="joy_cntl" type="" name="sim"/-->


 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="forest_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_linear" value="50" type="double"/>
  <param name="scale_angular" value="50" type="double"/>
  <param name="start_button" value="4" type="int"/>
  <node pkg="joy_cntl" type="joy_cntl_node" name="teleop"/>

  
  <node name="rossrial_python" pkg="rosserial_python" type="serial_node.py" output="screen">
  <param name="port" value="/dev/ttyACM0" />
  </node>  

</launch>
